images/prometheus/tests/main.tf

```diff
@@ -78,7 +78,48 @@ resource "helm_release" "kube-prometheus-stack" {
           tag        = data.oci_string.ref["operator"].pseudo_tag,
         }
       }
+      prometheus = {
+        enabled = true
+        prometheusSpec = {
+          image = {
+            registry   = data.oci_string.ref["core"].registry,
+            repository = data.oci_string.ref["core"].repo,
+            tag        = data.oci_string.ref["core"].pseudo_tag,
+          }
+        }
+      }
+      alertmanager = {
+        enabled = true
+        alertmanagerSpec = {
+          image = {
+            registry   = data.oci_string.ref["alertmanager"].registry,
+            repository = data.oci_string.ref["alertmanager"].repo,
+            tag        = data.oci_string.ref["alertmanager"].pseudo_tag,
+          }
+        }
+      }
```

Comment:
I'm not seeing these actually deploy.

I think we may want to deviate from our "generate values.yaml" approach for this one because the upstream version is 4000 lines long.